### PR TITLE
upgrade and use black 20

### PR DESCRIPTION
**Pull request by Jimexist**
seems that:
1. black now handles docstring more properly
2. it also folds params more aggressively

### CODE REVIEW

**Comment by Jimexist** [comment]
maybe not worth it...

**Comment by pascal-pfeiffer** [comment]
I see some issues in the Docstrings where `Args` and `Returns` are sometimes on different indentation levels.

**Comment by lucabergamini** [comment]
> * black now handles docstring more properly
> * it also folds params more aggressively

I'm fine with the first point (I like it!) but the second seems **really** aggressive. Is this the new default?

**Comment by Jimexist** [comment]
> > * black now handles docstring more properly
> > * it also folds params more aggressively
> 
> I'm fine with the first point (I like it!) but the second seems **really** aggressive. Is this the new default?

I think it's something with explicit trailing comma, if you remove them, black will no longer force line folding.

File: l5kit/l5kit/data/zarr_dataset.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 TL_FACE_DTYPE = [
     ("face_id", "<U16"),
     ("traffic_light_id", "<U16"),
-    ("traffic_light_face_status", np.float32, (len(TL_FACE_LABELS,))),
+    ("traffic_light_face_status", np.float32, (len(TL_FACE_LABELS),)),
```
Comment (Jimexist) [inline l5kit/l5kit/data/zarr_dataset.py:50]: i thought this should be a bug @lucabergamini

File: l5kit/Pipfile [hunk 2]
```diff
@@ -8,12 +8,12 @@ pytest = "*"
 mypy = "*"
 pytest-cov = "*"
 flake8 = "*"
-isort = "*"
+isort = "==5.6.2"
```
Comment (lucabergamini) [inline l5kit/Pipfile:11]: why pinning this?

**Review by lucabergamini — COMMENTED** [review]
minor :)

File: l5kit/Pipfile (hunk 2 above)
Comment (Jimexist) [reply l5kit/Pipfile:11]: like black, these things change and might silently break CI.

Comment (lucabergamini) [reply l5kit/Pipfile]: yes please, we have tests in place just to catch those issues (the ones without the lock) :)

**Review by lucabergamini — APPROVED** [review]
LGTM

**Pull request closed without merging**